3.10/graphs-enterprise-graphs-getting-started.md

```diff
@@ -113,35 +113,53 @@ After this step, the graph has been migrated.
 
 This example describes a scenario in which the collections names have changed,
 assuming that you have renamed `old_vertices` to `vertices`.
-For the vertex data this change is not relevant, the `_id` values is adjust automatically,
-so you can import the data again, and just target the new collection name:
+
+For the vertex data this change is not relevant, the `_id` values are adjusted
+automatically, so you can import the data again, and just target the new
+collection name:
 
 ```sh
 arangoimport --collection vertices --file docOutput/old_vertices.jsonl
 ```
 
 For the edges you need to apply more changes, as they need to be rewired.
+To make the change of vertex collection, you need to set
+`--overwrite-collection-prefix` to `true`.
 
-First thing, you have to remove the `_key` value as it is disallowed for
-EnterpriseGraphs.
-Second, because you have changed the name of the `_from` collection, you need
-to provide a `from-collection-prefix`. The same is true for the `_to` collection.
-
-Note that you can only change all vertices on `_from` respectively `_to`
-side with this mechanism. However, you can use different collections on `_from` and `_to`.
-
-Next, in order to make the change of vertex collection you need to
-allow `overwrite-collection-prefix`.
-If this flag is not set, only values without any given collection are changed.
-This is helpful if your data is not exported by ArangoDB in the first place.
-
-Now that you have everything together, run the following command:
+To migrate the graph and also change to new collection names, run the following
+command:
 
-```
+```sh
 arangoimport --collection edges --file docOutput/old_edges.jsonl --remove-attribute "_key" --from-collection-prefix "vertices" --to-collection-prefix "vertices" --overwrite-collection-prefix true
 ```
 
-After this step, the graph has been migrated and also changed to new collection names.
+Note that:
+- You have to remove the `_key` value as it is disallowed for EnterpriseGraphs.
+- Because you have changed the name of the `_from` collection, you need
+  to provide a `--from-collection-prefix`. The same is true for the `_to` collection,
+  so you need to provide a `--to-collection-prefix`.
```

Comment:
```suggestion
  so you also need to provide a `--to-collection-prefix`.
```

Reply:
I have no personal preference which of the above is better. I just think the "also that" part of the sentence is missing.

Reply:
Added an extra "also". Applied to all versions.